### DISCUSSION TITLE
更新日志 - 22.03.6 - 20240614

### DISCUSSION THREAD

**Discussion by jjm2473** [istoreos/istoreos#1639] [Announcements]
# 系统
1. 修复固件升级时会重置掉用户配置的软件源镜像的问题，并提高升级时清理overlayfs遗留文件的性能
2. 系统盘（根目录）满的时候，防止用户提交uci配置文件，以防破坏已有配置。如果在luci界面点击“保存应用”以后出现“Unknown Error”错误对话框，那可能就是磁盘满了，释放磁盘空间以后再点击“保存应用”即可再次尝试写入

# 设备
1. 新增支持 Radxa E20C（基于RK3528A），对此硬件有兴趣的话可关注 iStoreOS 公众号相关文章或 Radxa 官方资讯

**Comment by rayangl** [comment]
r2s安装更新后沙箱掉了，重启也不行，首页的沙箱按钮变成了开启沙箱，然后要格式化。这个能不格式化吗。

**Reply by jjm2473** [reply]
在挂载点里面可以配置，不过，既然你没改沙箱，那应该是因为文件系统错误无法挂载了，应该要修复下，然后重启就能回到沙箱

**Reply by rayangl** [reply]
> 在挂载点里面可以配置，不过，既然你没改沙箱，那应该是因为文件系统错误无法挂载了，应该要修复下，然后重启就能回到沙箱

这个要怎么修复呢？取下来，用diskgenius吗。
就是在挂载点找不到他

**Reply by rayangl** [reply]
> 在挂载点里面可以配置，不过，既然你没改沙箱，那应该是因为文件系统错误无法挂载了，应该要修复下，然后重启就能回到沙箱

![image](https://github.com/istoreos/istoreos/assets/71997121/44f1a3c6-86db-4222-adfb-17fff82d3bbc)
磁盘管理能看到他，点编辑，然后把它改成了mbr。但是好像没有生效

**Reply by jjm2473** [reply]
可能你的u盘已经坏了

**Comment by guoqiang881245** [comment]
我反应两个问题，1，docker安装portainer管理界面点击容器里面的统计提示错误，其他版本docker不会有这个错误。2，iStoreOS>服务>易有云文件管理>本地文件管理，公网https访问无法上传文件，只能用http访问才能上传。
![1718757629115](https://github.com/istoreos/istoreos/assets/84848631/ce3f2555-e7e3-4e61-b026-15c1b35b1cc3)
![微信截图_20240619084007](https://github.com/istoreos/istoreos/assets/84848631/59a854e6-e151-4c64-9e8e-8c5a2932f1fd)

**Reply by jjm2473** [reply]
docker这个问题其实跟docker本身没什么关系，只是看ps命令是不是兼容e参数而已，一般看容器，有些容器的ps支持e参数，有些不支持。

易有云的问题单独提个issue，方便找人跟进
